### PR TITLE
Rollback some changes from #12403

### PR DESCRIPTION
**Pull request by asmyasnikov**
### Changelog entry 

Rollback some changes from #12403

### Changelog category 

* Bugfix 
* Not for changelog (changelog entry is not required)

### Additional information

...

### CODE REVIEW

File: ydb/public/tools/lib/cmds/__init__.py [hunk 1]
```diff
@@ -320,7 +324,7 @@ def deploy(arguments):
     initialize_working_dir(arguments)
     recipe = Recipe(arguments)
 
-    if os.path.exists(recipe.metafile_path()):
+    if os.path.exists(recipe.metafile_path()) and enable_survive_restart()::
```
Comment (asmyasnikov) [inline ydb/public/tools/lib/cmds/__init__.py:327]:
```suggestion
    if os.path.exists(recipe.metafile_path()) and enable_survive_restart():
```

File: ydb/public/tools/lib/cmds/__init__.py [hunk 2]
```diff
@@ -260,6 +260,10 @@ def default_users():
     return {user: password}
 
 
+def enable_survive_restart():
+    return os.getenv('YDB_LOCAL_SURVIVE_RESTART') == 'true'
+
+
```
Comment (asmyasnikov) [inline ydb/public/tools/lib/cmds/__init__.py:266]:
```suggestion
```

File: ydb/public/tools/lib/cmds/__init__.py [hunk 3]
```diff
@@ -320,7 +324,7 @@ def deploy(arguments):
     initialize_working_dir(arguments)
     recipe = Recipe(arguments)
 
-    if os.path.exists(recipe.metafile_path()):
+    if os.path.exists(recipe.metafile_path()) and enable_survive_restart():
```
Comment (asmyasnikov) [inline ydb/public/tools/lib/cmds/__init__.py:327]:
```suggestion
    if os.path.exists(recipe.metafile_path()):
```

File: ydb/tests/library/harness/kikimr_config.py [hunk 4]
```diff
@@ -67,6 +67,9 @@ def _load_default_yaml(default_tablet_node_ids, ydb_domain_name, static_erasure,
     if isinstance(data, bytes):
         data = data.decode('utf-8')
     data = data.format(
+        ydb_result_rows_limit=os.getenv("YDB_KQP_RESULT_ROWS_LIMIT", 1000),
+        ydb_yql_syntax_version=os.getenv("YDB_YQL_SYNTAX_VERSION", "1"),
+        ydb_force_new_engine=os.getenv("YDB_KQP_FORCE_NEW_ENGINE", "true"),
```
Comment (asmyasnikov) [inline ydb/tests/library/harness/kikimr_config.py:72]:
```suggestion
```

File: ydb/tests/library/harness/resources/default_yaml.yml [hunk 5]
```diff
@@ -161,8 +162,14 @@ feature_flags:
   enable_mvcc_snapshot_reads: true
 kqpconfig:
   settings:
+  - name: "_ResultRowsLimit"
+    value: "{ydb_result_rows_limit}"
+  - name: "_KqpYqlSyntaxVersion"
+    value: "{ydb_yql_syntax_version}"
   - name: "_KqpAllowNewEngine"
     value: "true"
+  - name: "_KqpForceNewEngine"
```
Comment (gridnevvvit) [inline ydb/tests/library/harness/resources/default_yaml.yml:171]: вот тут забыл удалить.

File: ydb/tests/library/harness/resources/default_yaml.yml [hunk 6]
```diff
@@ -161,8 +162,14 @@ feature_flags:
...
+    value: "{ydb_force_new_engine}"
```
Comment (asmyasnikov) [inline ydb/tests/library/harness/resources/default_yaml.yml:172]:
```suggestion
```

**Review by gridnevvvit — APPROVED** [review]

**Pull request merged**